lib/test_prof/factory_prof/factory_bot_patch.rb

```diff
@@ -5,7 +5,19 @@ module FactoryProf
     # Wrap #run method with FactoryProf tracking
     module FactoryBotPatch
       def run(strategy = @strategy)
-        FactoryBuilders::FactoryBot.track(strategy, @name) { super }
+        variation = ''
+
+        if @traits || @overrides
+          unless @traits.empty?
+            variation += @traits.join('.').prepend('.')
+          end
+
+          unless @overrides.empty?
+            variation += @overrides.keys.to_s.gsub(/[\\":]/, '')
```

Comment:
The keys must be sorted, we shouldn't distinguish similar key sets.

Reply:
Done